webapp/src/components/sidebar_right/index.js

```diff
@@ -28,6 +28,7 @@ function mapDispatchToProps(dispatch) {
             bump,
             list,
             openRootModal,
+            setVisible,
```

Comment:
To be clear about the naming, it's fine to pass in the prop as `setVisible` because it is being passed specifically into the rhs component
```suggestion
            setVisible: setRhsVisible,
```